src/index.ts

```diff
@@ -42,7 +43,10 @@ const getUrlString = (url: string): string | null => {
   }
 }
 
-function remarkEmbedder({transformers, cache}: RemarkEmbedderOptions) {
+const remarkEmbedder: Plugin<[RemarkEmbedderOptions]> = ({
+  transformers,
+  cache,
+}: RemarkEmbedderOptions) => {
```

Comment:
A double declaration of `RemarkEmbedderOptions`, I think the second one should be able to be inferred somehow, no? 🤔

Reply:
Yeah, was testing something, but at the end is the same because they are indeed inferred, gonna remove it.